.gitlab-ci.yml

```diff
@@ -23,6 +23,7 @@
 stages:
   - check
   - test
+  - verify
```

Comment:
I think the whole new stage is a bit of an overkill for the only job. Even though it runs only ~9sec, it holds other jobs to start. Eventually, these seconds add up to minutes. Our stage names are already pretty much vague, esp with DAG here and there, so unless you plan to depend the following stage on the state of this one, just put it into `build`.

Reply:
Resolved [here](https://github.com/paritytech/substrate/pull/8519/commits/7ed876a3e2dd43398832311b1b80f8aa3a3d8f89).